MANIFESTO.md

```diff
@@ -11,7 +11,9 @@ The answer to this must be the collectivization of operational knowledge, just a
 From time to time, we can get mentally stuck and may need a sparring partner to think outside the box. Open Operations builds a community of practice to keep the barrier to entry low and create a thriving environment for comfortable exchange. Regardless of individual knowledge level, everyone contributes and enriches this community with their knowledge and individual views. Communities of practice connect through various channels serving both asynchronous and synchronous communication needs in the form of pair operating or on a larger scale to solve these challenges together. Therefore, we are building a network by encouraging organizations to foster and connect internal and external communities of practice.
 
 ## We share knowledge
-[...]
+Availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage and successfully operate (complex) technology.
```

Comment:
Imho we should not fall back to putting stuff in ( ). The manifest should be down to the point, short and easy to read.
Either it is part of the sentence or it is not. My suggestion:
"[...] and successfully operate complex technology".